main/adv_post.c

```diff
@@ -573,6 +582,7 @@ adv_post_generate_statistics_info(const str_buf_t* const p_reset_info)
     p_stat_info->reset_cnt    = reset_info_get_cnt();
     p_stat_info->p_reset_info = (NULL != p_reset_info) ? &p_reset_info->buf[0] : "";
 
+    // TODO: Do we need to use a separate nonce for stat?
```

Comment:
It is my understanding that nonces are used to protect against replay-attacks. Since the stat message is separate from data message, using same nonce should not be an issue.